### PR TITLE
add badge

### LINKED ISSUE #3: add badge on successful pour
display a QR code and/or token on the beer bot display to redeem the Matrix to Beer bridge 37c3 hub badge after successfully "authenticating" a pour.

probably simply update, adding another QR code:
https://github.com/yncyrydybyl/beerbot/blob/113c13e6e479f82d6207376effd7ec8b82046c2a/server/static/script.js#L17

### PR DESCRIPTION
**Pull request by HarHarLinks**
closes #3

### CODE REVIEW

**Comment by anoadragon453** [comment]
Note that the success message is only shown for 3 seconds:

https://github.com/yncyrydybyl/beerbot/blob/113c13e6e479f82d6207376effd7ec8b82046c2a/server/static/script.js#L141-L144

Which is probably not enough time for people to scan a QR code :) You may also want to put the instructions "Scan this QR code to earn your badge" underneath?

File: server/static/script.js [hunk 1]
```diff
@@ -152,5 +154,5 @@ function showError() {
     // Reset back to the home screen after one second.
     setTimeout(() => {
         showHome();
-    }, 1000);
+    }, 3000);
```
Comment (anoadragon453) [inline server/static/script.js:157]: What's the reason for upping the failure screen? In my mind, folks will want to get back to the default QR code as quickly as possible if they choose the wrong emoji on accident.

File: server/static/script.js [hunk 2]
```diff
@@ -138,10 +140,10 @@ async function pourBeer() {
     const grid = document.getElementById('mainContent');
     grid.innerHTML = SUCCESS_CONTENT;
 
-    // Reset back to the home screen after one second.
+    // Reset back to the home screen after pouring
     setTimeout(() => {
         showHome();
-    }, 3000);
+    }, 18000); // = POUR_TIME
```
Comment (anoadragon453) [inline server/static/script.js:146]: I think we can just couple these together - to ensure they don't get de-synced.

```suggestion
    }, POUR_TIME);
```

**Review by anoadragon453 — CHANGES_REQUESTED** [review]

File: server/static/script.js (hunk 1 above)
Comment (HarHarLinks) [reply server/static/script.js:157]: It seemed too short when I experienced how short 3 secs on success was

File: server/static/script.js (hunk 2 above)
Comment (HarHarLinks) [reply server/static/script.js:146]: POUR_TIME is in python though?

Comment (anoadragon453) [reply server/static/script.js:146]: Oh, d'oh, indeed it is.

I had the idea to pass it down in the response to `/pour`. The web UI could also show a countdown in seconds for the remaining pour time :)

Comment (HarHarLinks) [reply server/static/script.js:146]: yes, sounds nice.... uhh feel free to PR? :P

**Pull request merged**